checker/checker_breaking_property_test.go

```diff
@@ -540,9 +542,13 @@ func TestBreaking_WriteOnlyDeleteNonRequiredProperty(t *testing.T) {
 	require.NoError(t, err)
 	errs := checker.CheckBackwardCompatibility(checker.GetDefaultChecks(), d, osm)
 	require.NotEmpty(t, errs)
-	require.Len(t, errs, 1)
+	require.Len(t, errs, 3)
```

Comment:
Hi @blba and @tibulca 
Could you please review this PR and specifically this test which seems wrong (is it a breaking change or not)?
Thanks,
Reuven

Reply:
hi! 

I've looked at the history of `response-optional-property-removed` and it has always been a warning, we were probably not catching this because it wasn't added in default checks previously. 


- in this PR https://github.com/Tufin/oasdiff/pull/298 ResponseOptionalPropertyRemovedCheck (old name) was not removed, so it was never there.